src/views/Items/Item.js

```diff
@@ -0,0 +1,44 @@
+import React, {Component, PropTypes} from 'react';
+import classNames from 'classnames';
+import customPropTypes from '../../utils/customPropTypes';
+
+export default class Item extends Component {
+  static propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+    description: customPropTypes.mutuallyExclusive(['children']),
+    extra: PropTypes.node,
+    heading: PropTypes.node.isRequired,
+    image: PropTypes.node,
+    meta: PropTypes.node,
+  };
+
+  render() {
+    let heading = <div className='header'>{this.props.heading}</div>;
+    let meta = <div className='meta'>{this.props.meta}</div>;
+    let description = <div className='description'>{this.props.children || this.props.description}</div>;
+    let extra = <div className='extra'>{this.props.extra}</div>;
+    let content = (
+      <div className='middle aligned content'>
```

Comment:
Stardust should not add default style classes that are not configurable.  Perhaps exposing a `contentClass` prop would make sense.  We can then use `defaultProps` to set it to this value but the user could supply their own classes.  Use `classNames()` to merge the content classes the same as we do for className, adding `sd-content` and `content` classes automatically.